.changeset/eighty-mice-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/tag": major
```

Comment:
@273Do 
This is a minor fix and should be changed to a patch.

Reply:
Ok! 
Sorry for the many mistakes.